lib/matplotlib/ticker.py

```diff
@@ -1575,6 +1578,9 @@ def tick_values(self, vmin, vmax):
             subs = self._subs
 
         stride = 1
+        if not self.numticks > 1:
+            raise RuntimeError('The number of ticks must be greater than 1 '
+                               'for LogLocator.')
         while numdec / stride + 1 > self.numticks:
```

Comment:
Is numdec calculated correctly up above? It floors vmax, and ceils vmin, but down below, it is the other way around.